ihmc-footstep-planning/src/test/java/us/ihmc/footstepPlanning/log/FootstepPlannerLoggerTest.java

```diff
@@ -56,11 +54,43 @@ public void testFootstepLoggingSpeed()
       request.setHeightMapData(HeightMapMessageTools.unpackMessage(PlanarRegionToHeightMapConverter.convertFromPlanarRegionsToHeightMap(dataSet.getPlanarRegionsList())));
       request.setAssumeFlatGround(false);
       request.setPlanBodyPath(true);
+   }
 
-      planningModule.getFootstepPlannerParameters().setMaxStepZ(0.294);
-      planningModule.getFootstepPlannerParameters().setYawWeight(0.17);
-      planningModule.getFootstepPlannerParameters().setMaxZPenetrationOnValleyRegions(1.0);
+   /**
+    * This test was created to investigate a problem where logging didn't seem to be working, however, after more investigation it was because it was trying
+    * to be run on the Continuous Integration (CI) server.
+    * It's possible to set an environmental variable to run these classes "as if they were on the server" and
+    * that was happening here.
+    * This test isn't much computation and is nice to have lying around to prevent future change from breaking the logger
+    * This test likely won't run on CI because it's checking a hard coded path
+    */
+   @Disabled
+   @Test
+   public void testFootstepLoggingDefaultDirectory()
```

Comment:
Ensure that logs show up if using the default directory, more of a safety check then a critical test.